A66-otel-stats.md

```diff
@@ -0,0 +1,468 @@
+(with the above noted differences) allowing for receivers of the telemetry data
+to join the views from the two metrics for continuity.
+
+gRPC OpenCensus                  | gRPC OpenTelemetry
+-------------------------------- | ---------------------------------------------
+grpc.io/client/started_rpcs      | grpc.client.attempt.started
+grpc.io/client/completed_rpcs    | (Derivable from grpc.client.attempt.duration)
+grpc.io/client/roundtrip_latency | grpc.client.attempt.duration
+grpc.io/server/started_rpcs      | grpc.server.call.started
+grpc.io/server/completed_rpcs    | (Derivable from grpc.server.call.duration)
+grpc.io/server/server_latency    | grpc.server.call.duration
+
+#### Metrics with Nuanced Differences
+
+Unfortunately, the implementations of the gRPC OpenCensus spec in the various
+languages do not agree on the definition of the following size metrics. Go
+records uncompressed message bytes for the OpenCensus metric, while C++ and Java
+record the compressed message bytes. The OpenTelemetry spec proposed here calls
+for recording the compressed message bytes, resulting in an equivalence between
+the metrics definitions for C++ and Java, but not for Go.
+
+gRPC OpenCensus                       | gRPC OpenTelemetry
+------------------------------------- | ------------------
+grpc.io/client/sent_bytes_per_rpc     | grpc.client.attempt.sent_total_compressed_message_size
+grpc.io/client/received_bytes_per_rpc | grpc.client.attempt.rcvd_total_compressed_message_size
+grpc.io/server/sent_bytes_per_rpc     | grpc.server.call.sent_total_compressed_message_size
+grpc.io/server/received_bytes_per_rpc | grpc.server.call.rcvd_total_compressed_message_size
+
+#### OpenCensus Metrics not Initially Supported in OpenTelemetry
+
+There are some additional metrics defined in the gRPC OpenCensus spec and retry
+stats which we will not be supporting in the first iteration of the
+OpenTelemetry plugin. Some of these will eventually be accepted into the
+OpenTelemetry spec with the appropriate changes.
+
+*   Client Views
+    *   grpc.io/client/sent_messages_per_rpc
+    *   grpc.io/client/received_messages_per_rpc
+    *   grpc.io/client/server_latency
+    *   grpc.io/client/sent_messages_per_method
+    *   grpc.io/client/received_messages_per_method
+    *   grpc.io/client/sent_bytes_per_method
+    *   grpc.io/client/received_bytes_per_method
+*   Server Views
+    *   grpc.io/server/sent_messages_per_rpc
+    *   grpc.io/server/received_messages_per_rpc
+    *   grpc.io/server/sent_messages_per_method
+    *   grpc.io/server/received_messages_per_method
+    *   grpc.io/server/sent_bytes_per_method
+    *   grpc.io/server/received_bytes_per_method
+*   Retry Views
+    *   grpc.io/client/retries_per_call
+    *   grpc.io/client/retries
+    *   grpc.io/client/transparent_retries_per_call
+    *   grpc.io/client/transparent_retries
+    *   grpc.io/client/retry_delay_per_call
+
+## Migration Strategies
```

Comment:
This and its subsections should be indented one level more so that they are underneath the "Migration from OpenCensus" header.  This should be at the same level as the "Metric Schema Comparison" header.